docs/resources/cluster_azure.md

```diff
@@ -173,7 +173,7 @@ Optional:
 
 Required:
 
-- `azs` (Set of String) Availability zones for the machine pool.
+- `azs` (Set of String) Availability zones for the machine pool. Check if your region provides availability zones on [the Azure documentation](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-service-support#azure-regions-with-availability-zone-support).
```

Comment:
@nikchern should we provide a default value of ""? Otherwise it seems like the empty list will not prevent the user from avoiding the issue @addetz stumbled upon.

Reply:
@karl-cardenas-coding I think it's a good idea, let me know please if any help needed to add it.